app/graphql/query/singleActivity.graphql

```diff
@@ -0,0 +1,12 @@
+query activityCollection($user_id: BigInt!) {
+  activityCollection(filter: { id: { eq: $user_id } }) {
+    edges {
+      node {
+        id
+        uuid
+        trip_id
+        title
+      }
+    }
+  }
+}
```

Comment:
I think this file should be under app/activity directory if you do not have plan to use the other pages!

Reply:
Sorry I was just playing around with codegen. I didn't mean to push this file. I deleted it!